getting_started/workflow/best_practices/scenes_versus_scripts.rst

```diff
@@ -147,6 +147,63 @@ this way.
 
 On the downside, it also means having to use largely imperative programming.
 
+Performances of scripts vs PackedScenes
```

Comment:
I've personally always thought that the plural form, `Performances`, just sounds *incredibly* awkward in English. You only use it when referring to specific events of evaluating them, not when describing their performance in general.

I'd just drop the 's' to make it more clean. And change `scripts` to `Script`. If that's not enough of a reason, then you could also add more detail to the topic: `Comparing the performance of Script vs PackedScene`.